src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -79,25 +75,23 @@ For accounts on New Relic One pricing, some high-level billing information is di
 
   <Collapser
     id="user-count"
-    title="User count"
+    title="Full user count billing details"
   >
-    For accounts on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing), one billing factor is the monthly count of [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type), which you can see tracked [in the UI](#billing-usage-ui).
-
-    Here's an overview of how the monthly full user count is calculated:
+For accounts on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing), one billing factor is the monthly count of provisioned [full users](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type). For example, if you're on the Pro pricing tier and your organization has 100 full users present for the month of January, you'd be billed for 100 full users for that month. A full user counts as a billable user the moment they're provisioned (added to a New Relic organization), whether or not that user has logged into or used New Relic yet. 
 
-    1. The end-of-month full user count is based on an hourly resolution. For example, if a user was a full user for 180 hours out of a 30-day month (720 hours), that would be counted as 0.25 full users towards the total count.
-    2. Next, that end-of-month full user count is rounded to the nearest integer. For example, if the end-of-month full user count is 4.5, that account has 5 billable full users. If the end-of-month full user count is 4.45, that account has 4 billable full users.
-
-    Other calculation details:
+A user’s [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type) (basic user vs full user) is meant to be based on a user’s long-term expected New Relic responsibilities, not something often changed. For this reason, we have restrictions around how often a full user can be downgraded to a basic user: **a full user can downgrade to a basic user a maximum of two times in a rolling 12-month period.** If a full user has been changed to a basic user two times in that twelve-month period, that user won't be able to return to being a basic user until the start of the next 12-month period. To learn reasons for assigning one user type or another, see [Tips on assigning user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#decide-user-type).
```

Comment:
There's something about this first sentence that keeps throwing me. I'm trying to think through what it is...

The sentence is communicating these things:
1. A user type is meant to be a longterm, stable thing.
2. It's based on a user's responsibilities.
3. We don't intend people to change this. Often.

OK.... How about something more like this:

"A New Relic user type is a stable, long-term setting that describes what a user has access to, based on their responsibilities."

And then wouldn't it be a little more straightforward to say something like:

"Because user type is related to billing, we have restrictions on how often a full user can be downgraded to a basic user."

Thoughts?